src/Components/CommentCard.jsx

```diff
@@ -0,0 +1,12 @@
+function CommentCard({ comment }) {
+  return (
+    <section className="comment-card">
+      <h4>{comment.author}</h4>
+      <p>{comment.body}</p>
+      <p>{comment.votes}</p>
+      <p>{comment.created_at}</p>
```

Comment:
It would be better to have the timestamp formatted in a bit more of a human-readable way, than the default. Have a think about how you could manipulate the data yourself to do this or look into some packages that can do it for you 💡